### DISCUSSION TITLE
Configuring the vertical FOV for Ouster Sensors

### DISCUSSION THREAD

**Discussion by Samahu** [ouster-lidar/ouster-ros#18] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by suryanshsharma97** [comment]
Hi,
My name is Suryansh and I am using one of the ouster LIDAR sensor for my robot using ros. I am trying to configure the vertical FOV for the ouster sensor in ROS but havent been successful so far(found how to manipulate the horizontal FOV using azimuth_window), any tips on how to do that for the vertical FOV in ros?

**Reply by Samahu** [reply, marked as answer]
Hi Suryansh, I don't think it is possible to manipulate the vertical field of view through the software, but feel free to contact our [General Customer Support](https://ouster.atlassian.net/servicedesk/customer/portal/8/group/22/create/86) as they may be able to assist you further with such requests.

**Comment by suryanshsharma97** [comment]
Thanks a lot for your help, I doubt the customer support could do much on an already procured ouster sensor. Thanks a lot for your help, hope you have great week ahead.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Samahu** [comment]
@HadesReturns I have recently put a PR that reduces the vertical resolution of the #191. I understand that you want to reduce the upper and lower limits of the vertical FOV and not the resolution but following the same pattern shown in the PR you could reduce the FOV.
